planning/planning_topic_converter/src/path_to_trajectory.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "planning_topic_converter/path_to_trajectory.hpp"
+
+#include <motion_utils/trajectory/tmp_conversion.hpp>
+#include <tier4_autoware_utils/geometry/geometry.hpp>
+
+namespace planning_topic_converter
+{
+namespace
+{
+TrajectoryPoint convertToTrajectoryPoint(const PathPoint & point)
+{
+  TrajectoryPoint traj_point;
+  traj_point.pose = tier4_autoware_utils::getPose(point);
+  traj_point.longitudinal_velocity_mps = point.longitudinal_velocity_mps;
+  traj_point.lateral_velocity_mps = point.lateral_velocity_mps;
+  traj_point.heading_rate_rps = point.heading_rate_rps;
+  return traj_point;
+}
```

Comment:
(similar functions exist in other motion planner nodes, so better to use them as utils in the future)

Reply:
I totally agree with you. Create tickets to remind that task.